### ISSUE TITLE
direct support line protocol log data save into influxdb by fluentd ？

### ISSUE THREAD

**Issue by oss-jtyd** [influxdata/influxdb-plugin-fluent#25]
__Proposal:__
line protocol log data direct save into influxdb by fluentd

__Current behavior:__
use script analysis log file and than send to influxdb

__Desired behavior:__
line protocol log data direct save into influxdb by fluentd

__Alternatives considered:__

__Use case:__
log4j2.xml log message:
2021-04-27 11:18:26.395 INFO com349 sendOrder - mem,host=host1 used_percent=23.43234543 1556892576842902000
 how to save data into influxdb use Line protocol: mem,host=host1 used_percent=23.43234543 1556892576842902000

**Comment by bednar** [comment]
@oss-jtyd, thanks we will take a look

**Comment by bednar** [comment]
How looks like your parsed `fluentd` event? Am I right that your whole `LineProtocol` is parsed into one field in record?  Something like:

```json
time: 2021-04-27 11:18:26.395 +0900

record:
{
  "lineprotocol" : "mem,host=host1 used_percent=23.43234543 1556892576842902000"
}
```

**Comment by oss-jtyd** [comment]
yes， can be one field in record

**Comment by bednar** [comment]
I have prepared a fixed version within #31. Development version of the plugin will be available after #31 acceptance, stable version in regular release cycle.
